sdk/keyvault/Azure.Security.KeyVault.Administration/README.md

```diff
@@ -13,7 +13,7 @@ The Azure Key Vault administration library clients support administrative tasks
 Install the Azure Key Vault administration client library for .NET with [NuGet][nuget]:
 
 ```PowerShell
-dotnet add package Azure.Security.KeyVault.Administration --version 4.0.0-beta.1
+dotnet add package Azure.Security.KeyVault.Administration --version 4.0.0-beta.4
```

Comment:
this can be simplified to 
```suggestion
dotnet add package Azure.Security.KeyVault.Administration --prerelease
```

Reply:
Woahh

Reply:
I got the tip from @jsquire :)

Reply:
I'd like to, but that only works with .NET 5.0 SDK. I started a discussion internally.

Reply:
I used this for the most recent Tables release.. Didn't realize it was only supported in .NET 5

Reply:
For now I'll go with this, but we should definitely make a decision there and perhaps have the Prepare-Release.ps1 script update/verify it.

Reply:
Also tracked in #15876